src/styles/_common.scss

```diff
@@ -236,6 +236,9 @@ i {
     height: 160vw;
     @include viewport-unit(height, 160vw);
   }
+  @media (max-width: 400px) {
+    height: 100vh;
```

Comment:
Nooo! Don't. This cause mobile screens to jerk each time the URL bar hides and shows.